acquisitions/src/main/resources/thunderjet/mod-consortia/consortia-junit.feature

```diff
@@ -11,53 +11,63 @@ Feature: mod-consortia integration tests
       | 'mod-configuration'         |
     # following modules will also be enabled:
     # | 'mod-tags'                  |
+    # | 'mod-notes'                 |
     # | 'mod-users'                 |
-    # | 'folio_users'               |
-    # | 'mod-users-bl'              |
+    # | 'mod-email'                 |
+    # | 'mod-notify'                |
+    # | 'mod-pubsub'                |
+    # | 'mod-calendar'              |
     # | 'mod-consortia'             |
     # | 'mod-authtoken'             |
+    # | 'mod-feesfines'             |
+    # | 'mod-event-config'          |
+    # | 'mod-patron-blocks'         |
+    # | 'mod-template-engine'       |
+    # | 'mod-inventory-storage'     |
     # | 'mod-password-validator'    |
+    # | 'mod-circulation-storage'   |
```

Comment:
lets clear with  extra comment why we need these extra modules

Reply:
it is written in tenant-and-admin-setup.feature file, I will update it.